introduction.md

```diff
@@ -0,0 +1,2 @@
+You don’t have any verified emails. We recommend verifying at least one email.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.